### DISCUSSION TITLE
超讚的頻道

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#529] [General]
https://www.youtube.com/@Pyinvest/featured
講解很清楚 每部影片都很短又很清楚 很精闢的講解
而程式碼的部分 他會放在github 有學過本堂課 就知道可以fork 而後git clone抓下來
與本堂課比較相關的大概是 這個部分 https://www.youtube.com/watch?v=DeFIGJu6tbY&list=PLy7MS-q4l3xDYoR8MACYA3YyidUbEiz6j&ab_channel=PyInvest

https://www.youtube.com/@kurzgesagt
這是國外蠻知名的 科學頻道 超推

另外我個人的私心 不過應該每個人都知道 很有意義的頻道https://www.youtube.com/@MuseTW
我很推這部
https://www.youtube.com/watch?v=8ht4M4ZEoQ4&list=PL12UaAf_xzfq6GvHSYmj2VR1fGNcroSAU&ab_channel=Muse%E6%9C%A8%E6%A3%89%E8%8A%B1-TW

**Comment by todoroki0910** [comment]
看完這些頻道，學到最多東西的還是木棉花!

**Comment by brian431** [comment]
第二個頻道我也很喜歡，總是用風格非常特別的動畫來講解各種知識。

**Comment by awkward-willy** [comment]
現在網路上有各式各樣的學習資源方便大家快速上手各種技術，當然木棉花這種放鬆類型的也不少啦，至於怎麼對抗誘惑去看知識性頻道就又是另一回事了⋯⋯

**Comment by xCk27x** [comment]
看到文章最後面差點笑死XD

**Comment by Aerofe** [comment]
第一個之前有看過，那時候就感覺他講的很好，優質頻道

**Comment by williamyen0623** [comment]
現在自媒體蓬勃發展的時代，很多科技新知或技術都能從youtube上找到資源，但要篩選找到適合自己就需要花費一定的時間，謝謝版主分享給我們自己推薦的頻道，我會好好利用這些資源的

**Comment by HsuChen88** [comment]
第一個的知識性十足
第二個絕對是英文課最常看到的頻道

**Comment by Raywei0214** [comment]
那肯定不簡單。超讚的頻道的出現，必將帶領人類走向更高的巔峰。柯爾律治在過去曾經講過，講述生活折聲音沒有不和諧的。這是撼動人心的。
